lib/core/TruncateAsyncTask.js

```diff
@@ -9,11 +9,17 @@ var _execute = function(callback) {
 	
 	this.callback = callback;
 	var self = this;
-	fs.ftruncate(this.fd, this.fileSize, self.callback);
+  //support older versions of node
+  if (fs.ftruncate) {
+    fs.ftruncate(this.fd, this.fileSize, self.callback);
+  }
+  else {
+    fs.truncate(this.fd, this.fileSize, self.callback);
```

Comment:
I believe older versions of node don't have ftruncate, but do allow passing the file descriptor.  I actually didn't mean to create a PR against your repository, I meant to create it against the fork.   Happy to resubmit if you want the changes.